packages/perseus/src/widgets/interactive-graphs/graph-locked-labels-layer.tsx

```diff
@@ -37,22 +37,22 @@ export default function GraphLockedLabelsLayer(props: Props) {
         ) {
             return (
                 <React.Fragment key={i}>
-                    {figure.labels.map((label, j) => (
+                    {(figure.labels ?? []).map((label, j) => (
```

Comment:
This works, but could we just do... 
```suggestion
                    {figure.labels?.map((label, j) => (
```

Reply:
Huh... for some reason I was convinced that you couldn't use `map` on something that could be undefined 🤔 Good to know.